### DISCUSSION TITLE
Progress: set options on the fly

### DISCUSSION THREAD

**Discussion by sewnie** [ncruces/zenity#77] [Q&A] [answered]
Hi, i want to implement a simple progress dialog window, but it seems i cant quite specify a 'marquee' option. My usecase for this is to start the progressbar with no progress (eg. the bar itself jumping back and fourth, known as `Pulsate()`), and then later on start doing tasks and increment the bar.

**Comment by ncruces** [comment]
What are you using: the Go package, or the command line zenity? Which platform?

**Comment by sewnie** [comment]
Go, Linux, Wayland

**Comment by ncruces** [comment, marked as answer]
Sorry, memory fails me. This isn't possible, especially on Linux.

You can try this on the command line, run:
```sh
$ zenity --progress
```

Then update the progress bar by inputing a percentage on each line:
```sh
$ zenity --progress
10
20
30
```

It works! But if you add pulsate, it doesn't, anymore:
```sh
$ zenity --progress --pulsate
10
20
30
```

So there's no way I can make it work for `zenity`, and per policy described in [#1](https://github.com/ncruces/zenity/issues/11#issuecomment-1558924195), I left it out for other platforms where I could maybe make it work.

One way to make the dialog more useful in that case, which works only on Linux, is to add `zenity.TimeRemaining`.

**Reply by sewnie** [reply]
Thanks
